Exercises/NoModules/Chapter 10/Soln10_04.cpp

```diff
@@ -0,0 +1,49 @@
+// Exercise 10-4.cpp
+// Your very own interpretation of std::size()
+#include <iostream>
+#include <array>
+#include <vector>
+#include <span>
+
+// Look ma, no sizeof()!
+template <typename T, size_t N>
+size_t my_size(const T (&array)[N]) { return N; }
+
+// Overload with two other templates for std::vector<> and array<> 
+template <typename T>
+size_t my_size(const std::vector<T>& vector) { return vector.size(); }
+
+template <typename T, size_t N>
+size_t my_size(const std::array<T,N>& array) { return N; }  // or array.size();
+
+/*
+In stead of the latter two templates, you can create one template that would make my_size 
```

Comment:
"instead", 1 word

Reply:
Maybe I'll go with "in the stead of", which apparently is correct as well ;-)